index.html

```diff
@@ -42,187 +42,231 @@ <h4>I’am a Software Developer</h4>
+            </div>
+            <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            <div class="skill-frame-1 left-skill">
+                <img class="img-rectangle" src="/images/square.png" alt=""> <span class="img-title">Frameworks</span>
+            </div>
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>Rspec</li>
+                <li>CampyBara</li>
+                <li>Selenium</li>
+            </ul>
+            <div class="skill-frame-1 right-skill">
+                <img class="img-rectangle" src="/images/circle.png" alt=""> <span class="img-title">Skills</span>
+            </div>
+            <ul>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
         </div>
-        <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-        </ul>
-    </div>
-    <!-- Skill div End -->
-</section>
-<!-- About Section End -->
+        <!-- Skill div End -->
+    </section>
+    <!-- About Section End -->
+
+    <!-- Contact Form Start -->
+
+
+    <section>
+
+        <div id="maincontact">
+            <div class="contact">
+                <h2>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in
+                    touch.</h2>
+            </div>
+            <div class="form-contact">
+                <form action="#">
+                    <ul id="form-contact">
+                        <li><input type="text" class="name" maxlength="30" required name="name" placeholder="Full name">
+                        </li>
+                        <li><input type="email" class="email" name="email" required placeholder="Email Address"></li>
+                        <li><textarea class="txt-area" maxlength="500" name="txt-area" required id="txt-area" cols="30"
+                                rows="5" placeholder="Enter text here"></textarea></li>
+                    </ul>
+                    <div class="form">
+                        <button type="submit"> Get in touch</button>
+                    </div>
+                </form>
+                <footer>
+                    <div>
+                        <div id="c-line">
+                            <div></div>
+                        </div>
+                        <div id="s-icons">
+                            <img src="/images/social-media-icon.png" alt="">
+                        </div>
+                        <div class="l-line">
+                            <img src="/images/light.png" alt="">
+                        </div>
+                    </div>
+                </footer>
```

Comment:
- [ ] Kindly remove this section for it is not doing anything. Instead kindly use the `<footer>` tag to contain the contact form as indicated above.